class.jetpack-gutenberg.php

```diff
@@ -326,6 +326,20 @@ public static function get_jetpack_gutenberg_extensions_whitelist() {
 		return $preset_extensions;
 	}
 
+	/**
+	 * Returns a diff from a combined list of whitelisted extensions and extensions determined to be excluded
+	 *
+	 * @param  array $whitelisted_extensions An array of whitelisted extensions.
+	 *
+	 * @return array A list of blocks: eg [ 'publicize', 'markdown' ]
+	 */
+	public static function get_available_extensions( $whitelisted_extensions = null ) {
+		$exclusions             = get_option( 'jetpack_extensions_exclusions', array() );
+		$whitelisted_extensions = is_null( $whitelisted_extensions ) ? self::get_jetpack_gutenberg_extensions_whitelist() : $whitelisted_extensions;
```

Comment:
This is the line that could benefit from using an `array()` default value for the `whitelisted_extensions` argument.

```suggestion
		$whitelisted_extensions = ! $whitelisted_extensions ? self::get_jetpack_gutenberg_extensions_whitelist() : $whitelisted_extensions;
```

Alternatively, we could do it like this, which arguably demonstrates the intention better:

```
if( ! $whitelisted_extensions ) {
	$whitelisted_extensions = self::get_jetpack_gutenberg_extensions_whitelist();
}
```

Reply:
Agreed that the full `if` could be better. Though, I still think the `is_null` check for a default `null` value is a bit better.